examples/nlp/language_modeling/conf/megatron_t5_config_finetune_glue_mnli.yaml

```diff
@@ -0,0 +1,73 @@
+name: megatron_t5_glue
+
+trainer:
+  gpus: 2
+  num_nodes: 1
+  accelerator: ddp
+  precision: 16
+  logger: False # logger provided by exp_manager
+  checkpoint_callback: False
+  replace_sampler_ddp: False
+  max_epochs: 3
+  max_steps: null # consumed_samples = global_step * micro_batch_size * data_parallel_size * accumulate_grad_batches
+  log_every_n_steps: 10
+  val_check_interval: 300
+  accumulate_grad_batches: 2
+  gradient_clip_val: 1.0
+
+exp_manager:
+  explicit_log_dir: null
+  exp_dir: null
+  name: megatron_t5_glue
+  create_wandb_logger: False
+  wandb_logger_kwargs:
+    project: null
+    name: null
+  resume_if_exists: True
+  resume_ignore_no_checkpoint: True
+  create_checkpoint_callback: True
+  checkpoint_callback_params:
+    monitor: val_acc
+    save_top_k: 10
+    mode: max
+    always_save_nemo: False # TODO: add support
+    filename: 'megatron_t5--{val_acc:.3f}-{step}'
+    model_parallel_size: ${model.tensor_model_parallel_size}
+    save_best_model: True
+
+model:
+  restore_from_path: ??? # Path to a trained T5 .nemo file
+  tensor_model_parallel_size: 1
+
+  data:
+    train_ds:
+      task_name: 'mnli'
```

Comment:
I removed the `task_name` from the config. and added `task_name` to the data record so the model can handle multiple tasks simultaneously.